Sources/Networking/Responses/CustomerInfoResponse.swift

```diff
@@ -74,16 +78,34 @@ extension CustomerInfoResponse {
         var productIdentifier: String
         var purchaseDate: Date?
 
+        @IgnoreEncodable @IgnoreHashable
+        var rawData: [String: Any]
+
     }
 
 }
 
 // MARK: -
 
 extension CustomerInfoResponse.Subscriber: Codable, Hashable {}
-extension CustomerInfoResponse.Entitlement: Codable, Hashable {}
 extension CustomerInfoResponse.Subscription: Codable, Hashable {}
 
+extension CustomerInfoResponse.Entitlement: Hashable {}
+extension CustomerInfoResponse.Entitlement: Encodable {}
+extension CustomerInfoResponse.Entitlement: Decodable {
+
+    init(from decoder: Decoder) throws {
```

Comment:
maybe we could add a comment here explaining why we need to implement this method manually?

Reply:
Done.